fickling/pytorch.py

```diff
@@ -4,11 +4,18 @@
 from pathlib import Path
 from typing import Optional, Set
 
-import torch
-
 import fickling.polyglot
 from fickling.fickle import Pickled
 
+try:
+    import torch
+except ModuleNotFoundError:
+    raise ImportError(
+        "The 'torch' module is required for this functionality."
+        "PyTorch is now an optional dependency in Fickling."
+        "Please use pip install '.[torch]'"
```

Comment:
Shouldn't this be `pip install fickling[torch]`? I'm assuming people won't install by cloning the repo but do it from PyPI directly so `.[torch]` doesn't apply to them.